tests/core/types/test_spend_bundle.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from blspy import G2Element
+
+from chia.types.spend_bundle import SpendBundle
+
+
+BLANK_SPEND_BUNDLE = SpendBundle(coin_spends=[], aggregated_signature=G2Element())
+NULL_SIGNATURE = "0xc" + "0" * 191
+
+
+class TestStructStream:
```

Comment:
The test would become deprecated with the legacy name being deprecated, but i feel like there should be a test to make sure that for now `.to_json_dict` returns a spendbundle using "coin_solutions"

Reply:
Yes, good idea.